homework/index.js

```diff
@@ -29,22 +29,51 @@
     return elem;
   }
 
+  function changeDateTimeFormat(dateTime) {
+    const timeFormat = new Date(dateTime);
+    return timeFormat.toLocaleString();
+  }
+  function makeTable(table, header, value) {
+    const tr = createAndAppend('tr', table, { class: 'tr' });
+    createAndAppend('th', tr, { text: header, class: 'keys' });
+    createAndAppend('td', tr, { text: value, class: 'values' });
+    return tr;
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    const table = createAndAppend('table', ul, { class: 'table' });
```

Comment:
This generates invalid HTML. A `ul` can only have  `li` tags as its children. You need to create an `li` tag and append it to the `ul`. Then, create the `table` tag and append it to the `li`.